cmd/refresh.go

```diff
@@ -48,10 +48,8 @@ func newRefreshCmd() *cobra.Command {
 		Short: "Refresh the resources in a stack",
 		Long: "Refresh the resources in a stack.\n" +
 			"\n" +
-			"This command compares the current stack's resource state with the state known to exist in\n" +
-			"the actual cloud provider. Any such changes are adopted into the current stack. Note that if\n" +
-			"the program text isn't updated accordingly, subsequent updates may still appear to be out of\n" +
-			"synch with respect to the cloud provider's source of truth.\n" +
+			"This command resets the 'Pulumi.<stack name>.yaml' file to match the state snapshot copy from the last\n" +
+			"'pulumi up' command.\n" +
```

Comment:
This doesn't looks correct.

Reply:
I wrote this based on Patrick's input.  If it is not correct, then at least the current text is confusing and we should write here whatever is correct.  How about I circle back to Patrick again and check this?